### PR TITLE
Add kinesis sink 3.0 docs.

### PR DESCRIPTION
**Pull request by shibd**

### CODE REVIEW

**Comment by momo-jun** [comment]
@shibd, does it make sense to use a google doc to facilitate the review?

**Comment by shibd** [comment]
> @shibd, does it make sense to use a Google doc to facilitate the review? It would be efficient to comment and suggest changes there.

But using Google Docs would lose our review process.

**Comment by momo-jun** [comment]
> But using Google Docs would lose our review process.

Just like the SN connector docs we are improving/reviewing, using the Google doc can host the initial review to get a solid draft. Then we can start to use a pull request to host the final review and signoff. We can attach the G-doc link in the PR, so that all the historical comments can be traced back.

**Comment by shibd** [comment]
Sure, I put this docs on [here](https://docs.google.com/document/d/1KffpWw1-W8zyFCBymS8pSTDWKyZz_jFuwaffFbR2_zM)

File: connectors/kinesis-sink/3.0/kinesis-sink.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+- [Create a Connector with Function mesh](https://functionmesh.io/docs/connectors/run-connector).
+Using this way requires you to set the docker image. You can choose the version you want to launch from [here](https://hub.docker.com/r/streamnative/pulsar-io-kinesis).
+
+No matter how you create an AWS Kinesis sink connector, the minimum configuration contains the following parameters.
+```yaml
+configs:
+awsRegion: "Your aws kinesis region"
+awsKinesisStreamName: "Your kinesis stream name"
+awsCredentialPluginParam: "{\"accessKey\":\"Your access key\",\"secretKey\":\"Your secret key\"}"
+```
```
Comment (RobertIndie) [inline connectors/kinesis-sink/3.0/kinesis-sink.md:56]: How about we guide users here to get the awsRegion and awsCredentialPluginParam? We could just copy the description of `awsRegion` and `awsCredentialPluginParam` from the `Configuration Properties` chapter. Or we could link to them.

Comment (shibd) [reply connectors/kinesis-sink/3.0/kinesis-sink.md:56]: 1. We covered how to get `AccessKey` and `SecretKey ` in **Prerequisites**.
2. `awsRegion` is a very easy concept to understand, and we've covered it in the next line that users can go to **Configuration Properties** for a more detailed explanation
> If you want to configure more parameters, see [Configuration Properties](https://github.com/streamnative/pulsar-hub/diffs/0?base_sha=7e8c2208e934f8bab9ee68fa486b15f719ca2312&branch=improve_kinesis_sink&commentable=true&name=improve_kinesis_sink&pull_number=290&qualified_name=refs%2Fheads%2Fimprove_kinesis_sink&sha1=7e8c2208e934f8bab9ee68fa486b15f719ca2312&sha2=b24e8bfbb5ff06fcf8d17626911973aaca900b4d&short_path=eeb84b4&unchanged=expanded&w=false#configuration-properties) for reference.

Comment (RobertIndie) [reply connectors/kinesis-sink/3.0/kinesis-sink.md:56]: Make sense to me.

**Review by RobertIndie — APPROVED** [review]

**Pull request merged**